### PR TITLE
Release 1.2.0

### LINKED ISSUE #2583: Streamlink 1.2.0
I think we should publish a new release. There are a lot of changes and crucial plugin fixes since the last one in April.

Any open and unresolved blockers? Anything that should be merged first? Does anybody want to write a short changelog? Optional though, as we include the git shortlog anyway.

### PR DESCRIPTION
**Pull request by back-to**
closes #2583

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,96 @@
 # Changelog
 
+## streamlink 1.2.0 (2019-08-18)
+
+Here are the changes to this months release!
```
Comment (bastimeyer) [inline CHANGELOG.md]: > Here are the changes of this month's release!

or

> Here are the changes of the release of this month!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+
+- Multiple plugin fixes
+- Fixed single hyphen params at the beginning of --player-args (#2333)
+- `--http-proxy` will set the default value of `--https-proxy` to same as `--http-proxy`. (#2536)
+- DASH Streams will handle headers correctly (#2545)
+- the timestamp for FFMPEGMuxer streams will start with zero (#2559)
+
+```text
```
Comment (bastimeyer) [inline CHANGELOG.md]: Please use two empty lines for consistency.

File: CHANGELOG.md (hunk 1 above)
Comment (beardypig) [reply CHANGELOG.md]: I might say 
> Here are the changes for this month's release

**Comment by back-to** [comment]
Done
https://github.com/streamlink/streamlink/releases/tag/1.2.0

**Comment by beardypig** [comment]
Looks good. Thanks @back-to

**Pull request merged**